src/main/java/dev/jbang/util/JavaUtil.java

```diff
@@ -96,6 +96,8 @@ private static int determineJavaVersion(Path javaCmd) {
 	public static Path getJdkHome() {
 		if (getenv("JAVA_HOME") != null) {
 			return Paths.get(getenv("JAVA_HOME"));
+		} else if (System.getProperty("java.home") != null) {
```

Comment:
looks right but does it fix something related to this issue of --repos ?

Reply:
No, this is related to me needing to be able to run the tests :-)

After upgrading to Fedora 36 the default Java is a JRE. I could fix that but I thought it was just better to make the tests work in these cases too.